src/graphrag_toolkit/storage/falkordb_graph_store.py

```
@@ -0,0 +1,133 @@
+            try:
+                self._driver = falkordb.FalkorDB(
+                        host=self.host,
+                        port=self.port,
+                        username=self.username,
+                        password=self.password,
+                        ssl=self.ssl,
+                    )
+                self._client = self._driver.select_graph(self.database)
+            except Exception as e:
+                logger.error(f"Failed to connect to FalkorDB: {e}")
+                raise ConnectionError(f"Could not establish connection to FalkorDB: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)